projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -131,8 +131,8 @@
     </div>
     <!-- Actions -->
     <div *ngIf="content.actions && !editMode" class="cx-card-actions">
-      <div *ngFor="let action of content.actions">
-        <div>
```

Comment:
It might just be the way the change is displayed. But have we removed a nested `div`?

Reply:
Ok, can we please put it back? This changes the DOM structure which is a breaking change.